### DISCUSSION TITLE
units library

### DISCUSSION THREAD

**Discussion by xiaohuagong** [humanpred/pknca#224] [Q&A] [answered]
In order to achieve automatic units conversion (https://billdenney.github.io/pknca/articles/v07-unit-conversion.html), a units library is needed. There seem quite some such libraries. Is there a shortlist for recommendation?

**Comment by billdenney** [comment, marked as answer]
It is specifically **the** units library: https://cran.r-project.org/web/packages/units/

I believe that if you install PKNCA with `install.packages("PKNCA")`, `units` will be automatically installed.
